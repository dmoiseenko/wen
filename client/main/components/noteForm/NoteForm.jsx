import React from 'react';
import { Field } from 'redux-form';

import Input from '../../../common/components/input/Input';
import { textField } from '../../core/note.form.constant';
import NewNoteContainer from './NewNote.container';


export default function NoteForm() {
  return (
    <form className="container">
      <Field
        name={textField}
        component={Input}
        placeholder="note text"
        type="text"
        iconClassName="fa fa-pencil"
      />
      <div className="field">
        <NewNoteContainer />
      </div>
    </form>
  );
}
