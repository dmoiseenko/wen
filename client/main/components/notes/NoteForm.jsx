import React from 'react';
import { Field } from 'redux-form';

import Input from '../../../common/components/input/Input';
import { textField } from '../../core/note.form.constant';
import NewNoteContainer from '../notes/NewNote.container';


export default function NoteForm() {
  return (
    <form className="box">
      <Field
        name={textField}
        component={Input}
        placeholder="note text"
        label="new note"
        type="text"
      />
      <div className="field">
        <NewNoteContainer />
      </div>
    </form>
  );
}
