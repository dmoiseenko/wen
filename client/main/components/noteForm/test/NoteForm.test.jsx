import React from 'react';
import { shallow } from 'enzyme';

import NoteForm from '../NoteForm';


it('should render', () => {
  const wrapper = shallow(
    <NoteForm />
  );

  expect(wrapper.node).toMatchSnapshot();
});
