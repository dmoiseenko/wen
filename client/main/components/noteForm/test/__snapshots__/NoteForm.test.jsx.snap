// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<form
  className="container"
>
  <Field
    component={[Function]}
    iconClassName="fa fa-pencil"
    name="textField"
    placeholder="note text"
    type="text"
  />
  <div
    className="field"
  >
    <Connect(Apollo(mapProps(NewNote))) />
  </div>
</form>
`;
